client/row.go

```diff
@@ -58,9 +58,21 @@ func (c *GrpcClient) CreateCollectionByRow(ctx context.Context, row entity.Row,
 	return nil
 }
 
-// InsertByRows insert by rows
+// InsertByRows DEPRECATED, use  InsertRows instead
 func (c *GrpcClient) InsertByRows(ctx context.Context, collName string, partitionName string,
```

Comment:
Prefer to remove this as this version has not been published

Reply:
`InsertByRows` has already in published version in several releases